omnigibson/utils/config_utils.py

```diff
@@ -2,6 +2,7 @@
 import json
 import os
 
+import numpy as np
```

Comment:
Why is this necessary?

Reply:
sorry, not needed (was part of WIP) -- removed!